node/runtime/src/constants.rs

```diff
@@ -18,21 +18,25 @@
 
 /// Money matters.
 pub mod currency {
-    use node_primitives::Balance;
+	use node_primitives::Balance;
 
-    pub const MILLICENTS: Balance = 1_000_000_000;
-    pub const CENTS: Balance = 1_000 * MILLICENTS;    // assume this is worth about a cent.
-    pub const DOLLARS: Balance = 100 * CENTS;
+	pub const MILLICENTS: Balance = 1_000_000_000;
+	pub const CENTS: Balance = 1_000 * MILLICENTS;    // assume this is worth about a cent.
+	pub const DOLLARS: Balance = 100 * CENTS;
 }
 
 /// Time.
 pub mod time {
-    use node_primitives::Moment;
+	use node_primitives::Moment;
 
-    pub const SECS_PER_BLOCK: Moment = 6;
-    pub const MINUTES: Moment = 60 / SECS_PER_BLOCK;
-    pub const HOURS: Moment = MINUTES * 60;
-    pub const DAYS: Moment = HOURS * 24;
+	pub const MILLISECS_PER_BLOCK: Moment = 6000;
+	pub const SECS_PER_BLOCK: Moment = MILLISECS_PER_BLOCK / 1000;
+	pub const SLOT_DURATION: Moment = 6000;
```

Comment:
I assume this will not always be same as block time right? Given that, a short doc for at the least the two and how they are different would be helpful.

PS. Thanks for the _tab-ification_.

Reply:
Yes, you are right. If you don't mind I'll add some docs in the follow-up PR (#3211) where I set this correctly for BABE.